docs/reference/commandline/run.md

```diff
@@ -544,13 +544,13 @@ retrieve the container's ID once the container has finished running.
 
 ```bash
 $ docker run --device=/dev/sdc:/dev/xvdc \
-             --device=/dev/sdd --device=/dev/zero:/dev/nulo \
+             --device=/dev/sdd --device=/dev/zero:/dev/null \
```

Comment:
I don't think this change is correct; the example is not intending to "override" `/dev/null` in the container. Perhaps changing it to `/dev/foo` (e.g.) would be better if this confuses people though